libs/dh/metering-point/feature-overview/src/components/dh-metering-point-master-data.component.ts

```diff
@@ -162,10 +165,24 @@ import { DhMeteringPointHighlightsComponent } from './dh-metering-point-highligh
         />
 
         @if (maybeRelatedMeteringPoints()) {
-          <dh-related-metering-points
-            [relatedMeteringPoints]="relatedMeteringPoints()"
-            [meteringPointId]="meteringPointId()"
-          />
+          @defer (on idle) {
+            <watt-card>
+              <watt-card-title>
+                <h3>{{ 'meteringPoint.relatedMeteringPointsTitle' | transloco }}</h3>
+              </watt-card-title>
+              <dh-result
+                [loading]="loadingRelatedMeteringPoints()"
+                [hasError]="hasErrorRelatedMeteringPoints()"
+                variant="compact"
+                loadingText="{{ 'meteringPoint.relatedMeteringPointsLoading' | transloco }}"
```

Comment:
nitpick: just remembered that the default empty text is different from that in Figma. Should we expand the component to support custom empty text, or the default is good enough. UX can probably answer that one but I also suspect that they would like custom text too :)

Reply:
Good catch